src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  width: 7 * ($day-size + $gap) - $gap + 2 * $calendar-padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $calendar-padding;
+
+  &__day {
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid #000;
+    background-color: #eee;
+    transition-duration: 0.5s;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @for $i from 1 through $default-month-length {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+      }
```

Comment:
Make sure that the variable $default-month-length is defined in your variables file and contains the correct number of days for the default month length. This is not explicitly stated as an error, but it's important for the functionality of the calendar.